lib/ddtrace/contrib/faraday/middleware.rb

```diff
@@ -28,7 +28,7 @@ def initialize(app, options = {})
         def call(env)
           dd_pin.tracer.trace(SERVICE) do |span|
             annotate!(span, env)
-            propagate!(span, env) if options[:distributed_tracing]
+            propagate!(span, env) if options[:distributed_tracing] && dd_pin.tracer.enabled
```

Comment:
I think the test should wait for the merge of #229 and more precisely the introduction of WebMock https://github.com/DataDog/dd-trace-rb/pull/229/files#diff-b06e29a984e208fededfd0c6e60ca9ecR41 which could be very useful and straightforward to write a simple test for this.

Reply:
What test should we write?

Reply:
See https://github.com/DataDog/dd-trace-rb/pull/254/commits/70f018f18dba8162cd36bedfbea1f2dfc60776ac